### PR TITLE
refactor: Rename util class and move common methods to util class

### PR DESCRIPTION
**Pull request by o-kopysov**
# Pull Request

## Description

Refactoring of the Util class:
- move common methods from other classes to util
- rename the util class based on the functionality
- clean unused lines from unit tests

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] Code cleanup/refactoring
- [ ] Documentation update
- [ ] This change requires a documentation update
- [ ] CI system update
- [X] Test Coverage update

## Testing

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration.

**Test Configuration**:
* Java: v17
* LPVS Release: v1.5.0

# Checklist:

- [X] My code follows the style guidelines of this project
- [X] My code meets the required code coverage for lines (90% and above)
- [X] My code meets the required code coverage for branches (80% and above)
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/main/java/com/lpvs/util/LPVSPayloadUtil.java [hunk 1]
```diff
@@ -24,7 +28,29 @@
...
+    /**
+     * Creates an InputStreamReader for the given process, handling UTF-8 encoding.
+     *
+     * @param inputStream The input stream to read from.
+     * @return An InputStreamReader for the error stream of the process.
+     * @throws UnsupportedEncodingException If UTF-8 encoding is not supported.
+     */
+    public static InputStreamReader createInputStreamReader(InputStream inputStream)
+            throws UnsupportedEncodingException {
+        return new InputStreamReader(inputStream, "UTF-8");
```
Comment (tiokim) [inline src/main/java/com/lpvs/util/LPVSPayloadUtil.java:42]: Is returning the `error` stream correct? 
I think either description or code implementation should be modified.

**Review by tiokim — APPROVED** [review]
Thank you for opening the PR. 
Overall it looks great, but take a look at my comment.

File: src/main/java/com/lpvs/util/LPVSPayloadUtil.java (hunk 1 above)
Comment (o-kopysov) [reply src/main/java/com/lpvs/util/LPVSPayloadUtil.java:42]: Thanks. The description is outdated.

**Review by tdrozdovsky — APPROVED** [review]

**Review by t-naumenko — APPROVED** [review]
LGTM

**Pull request merged**